webapp/app/controllers/HomeController.scala

```diff
@@ -183,14 +193,18 @@ class HomeController @Inject()(cc: ControllerComponents) extends AbstractControl
       e.trigger +: argTriggers.toSeq
     }
     // collect arguments as text bound mentions
-    val entities = for {
-      e <- events ++ relations
-      a <- e.arguments.values.flatten
-    } yield a match {
-      case m: TextBoundMention => m
-      case m: RelationMention => ???
-      case m: EventMention => m.trigger
-    }
+//    val entities = for {
```

Comment:
Do you want to erase these commented-out lines?

Reply:
It looks deleted in the latest version of the branch? Or am I looking someplace wrong?

Reply:
Lines 196 through 203 in this file have some old code commented out. I think these should be removed before merging.

Reply:
Didn't this commit take care of that? https://github.com/clulab/reach/commit/f779d1ac7eab8a640cffdce7523e6746331ba083

Reply:
Yes. My bad. All good then!

Reply:
Great! Thank you!